torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -789,10 +807,22 @@ def _generate_comparision_values(self, input_info: Dict, weight_info: Dict) -> D
             weight_ratio = self._calculate_range_ratio(weight_info[module_fqn], self.WEIGHT_STR, module_fqn)
             input_ratio = self._calculate_range_ratio(input_info[module_fqn], self.INPUT_STR, module_fqn)
 
+            # if mismatched size, because of grouping, we want to replicate weight enough times
+            weight_channels = len(weight_ratio)
+            input_channels = len(input_ratio)
+            if weight_channels != input_channels:
+                # we try to replicate
+                if input_channels % weight_channels == 0:
+                    # get replication factor
+                    rep_factor: int = input_channels // weight_channels
+                    weight_ratio = weight_ratio.repeat(rep_factor)
```

Comment:
which dimension is this repeating at? I think we might need some comments since this op is not very familiar to most of people